src/tract.h

```diff
@@ -109,7 +109,7 @@ extern int         is_readapp (ITEMSET *iset, FILE *file);
 extern int         is_read    (ITEMSET *iset, FILE *file);
 
 extern int         is_recode  (ITEMSET *iset, int minfrq, int dir,
-                               int *map);
+                               int *map,  int mode, int fullS);
```

Comment:
/*Makh2018; Change is_recode function to remove full support items in case of generator itemsets mining */